lua/entities/gmod_wire_expression2/core/bone.lua

```diff
@@ -108,7 +108,7 @@ end
 --- Returns an array containing all of <this>'s bones. This array's first element has the index 0!
 e2function array entity:bones()
 	if not IsValid(this) then return { } end
-	return GetBones(this)
+	return table.Copy(GetBones(this))
```

Comment:
I forget, are tables like this allowed in e2? I thought they need an array of types and such, or is it just arrays that do that?

Reply:
Nope, E2 arrays are pure Lua tables. E2 tables get that luxury.